ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -97,12 +96,14 @@ Logout From RHODS Dashboard
 
 Wait For RHODS Dashboard To Load
     [Arguments]  ${dashboard_title}="${ODH_DASHBOARD_PROJECT_NAME}"    ${wait_for_cards}=${TRUE}
-    ...          ${expected_page}=Enabled    ${timeout}=60
+    ...          ${expected_page}=None    ${timeout}=60
     ${half_timeout}=   Evaluate    int(${timeout}) / 2
     Wait For Condition    return document.title == ${dashboard_title}    timeout=${half_timeout}
     Wait Until Page Contains Element    xpath:${RHODS_LOGO_XPATH}    timeout=${half_timeout}
     IF    "${expected_page}" != "${NONE}"
         Wait For Dashboard Page Title    ${expected_page}    timeout=${timeout}
+    ELSE
+        Wait Until Page Contains Element    //div[@data-testid="home-page"]    timeout=${half_timeout}
     END
```

Comment:
would negate the condition and flip the branches

```suggestion
    IF    "${expected_page}" == "${NONE}"
        Wait Until Page Contains Element    //div[@data-testid="home-page"]    timeout=${half_timeout}
    ELSE
        Wait For Dashboard Page Title    ${expected_page}    timeout=${timeout}
    END
```

Reply:
good point ,updated :)